topics/open-policy-agent/index.md

```diff
@@ -0,0 +1,20 @@
+---
+aliases: opa, openpolicyagent
+display_name: Open Policy Agent
+github_url: https://github.com/open-policy-agent/opa
+logo: opa.png
+related: policy-as-code, policy, authorization, open-source-policy
+short_description: OPA is an open source, general-purpose policy engine that unifies policy enforcement across the stack.
+topic: open-policy-agent
+url: https://www.openpolicyagent.org/
+---
+The Open Policy Agent (OPA, pronounced “oh-pa”) is an open source, general-purpose policy engine that unifies policy enforcement across the stack. OPA provides a high-level declarative language that lets you specify policy as code and simple APIs to offload policy decision-making from your software. You can use OPA to enforce policies in microservices, Kubernetes, CI/CD pipelines, API gateways, and more.
+
+### Community & Support
+- [Website](https://www.openpolicyagent.org/)
+- [GitHub](https://github.com/open-policy-agent/)
+- [Documentation](https://www.openpolicyagent.org/docs/latest/)
+
+**What is OPA**
+
+[//youtube-embed-unfurl//]: # (ijQdHy4XQCU)
```

Comment:
I'm not seeing anything in the rendered Markdown for this section. What are you intending to include?

Reply:
I was trying to replicate the pattern on the [actions topic](https://raw.githubusercontent.com/github/explore/main/topics/actions/index.md) (see the bottom line). But it doesn't seem to be rendering there either.  It's meant to be an introductory video to OPA: https://www.youtube.com/watch?v=ijQdHy4XQCU&t.  I'm fine with removing it entirely, or if there's a better way to render video, we can do that, or maybe just link to it?  Just let me know.

Reply:
Oh, interesting! Let's give it a shot!